frontend/src/metabase/selectors/embed.ts

```diff
@@ -1,7 +1,7 @@
 import { isWithinIframe } from "metabase/lib/dom";
 import { State } from "metabase-types/store";
 
-export const getIsEmbedded = () => {
+export const getIsEmbedded = (state: State) => {
   return isWithinIframe();
 };
```

Comment:
Have to explicitly mention `State` here. Otherwise, `reselect` selectors using `getIsEmbedded` are going to confuse TypeScript :(